src/main/java/picard/util/LiftoverUtils.java

```diff
@@ -0,0 +1,255 @@
+package picard.util;
```

Comment:
needs license

Reply:
ok.  i see many of these classes are missing licences?  SortVcf being an example, and LiftoverVcf (pre-dated my PR), also seems to lack one?

Reply:
Please add to your new files in this PR. I'll open an issue to add licenses to the ones that miss it. thanks.

Reply:
ok